website/client/src/components/auth/reset.vue

```diff
@@ -0,0 +1,193 @@
+<template>
+  <form
+    id="reset-password-set-new-one-form"
+    @submit.prevent="resetPasswordSetNewOneLink"
+    @keyup.enter="resetPasswordSetNewOneLink"
+  >
+    <div class="form-group">
+      <label
+        v-once
+        for="passwordInput"
+      >{{ $t('newPass') }}</label>
+      <input
+        id="passwordInput"
+        v-model="password"
+        class="form-control input-with-error"
+        type="password"
+        :placeholder="$t('password')"
+        :class="{'input-invalid': passwordInvalid, 'input-valid': passwordValid}"
+      >
+      <div
+        v-if="passwordInvalid"
+        class="input-error"
+      >
+        {{ $t('minPasswordLength') }}
+      </div>
+    </div>
+    <div class="form-group">
+      <label
+        v-once
+        for="confirmPasswordInput"
+      >{{ $t('confirmPass') }}</label>
+      <input
+        id="confirmPasswordInput"
+        v-model="passwordConfirm"
+        class="form-control input-with-error"
+        type="password"
+        :placeholder="$t('confirmPasswordPlaceholder')"
+        :class="{'input-invalid': passwordConfirmInvalid, 'input-valid': passwordConfirmValid}"
+      >
+      <div
+        v-if="passwordConfirmInvalid"
+        class="input-error"
+      >
+        {{ $t('passwordConfirmationMatch') }}
+      </div>
+    </div>
+    <div class="text-center">
+      <div
+        class="btn btn-info"
+        :enabled="!resetPasswordSetNewOneData.hasError"
+        @click="resetPasswordSetNewOneLin"
+      >
+        {{ $t('setNewPass') }}
+      </div>
+    </div>
+  </form>
+</template>
+
+<style lang="scss" scoped>
+  @import '~@/assets/scss/colors.scss';
+  #reset-password-set-new-one-form {
+    margin: 0 auto;
+    width: 40em;
+    padding-top: 5em;
+    padding-bottom: 4em;
+    position: relative;
+    z-index: 1;
+
+    label {
+      color: $white;
+      font-weight: bold;
+    }
+
+    input {
+      margin-bottom: 2em;
+      border-radius: 2px;
+      background-color: #432874;
+      border-color: transparent;
+      height: 50px;
+      color: $white;
+    }
+
+    .input-with-error.input-invalid {
+      margin-bottom: 0.5em;
+    }
+
+    #confirmPasswordInput + .input-error {
+      margin-bottom: 2em;
+    }
+
+    .form-text {
+      font-size: 14px;
+      color: $white;
+    }
+    ::-webkit-input-placeholder { /* Chrome/Opera/Safari */
+      color: $purple-400;
+    }
+    ::-moz-placeholder { /* Firefox 19+ */
+      color: $purple-400;
+    }
+    :-ms-input-placeholder { /* IE 10+ */
+      color: $purple-400;
+    }
+    :-moz-placeholder { /* Firefox 18- */
+      color: $purple-400;
+    }
+    ::placeholder { //  Standard browsers
```

Comment:
This is not specific to this PR but in all places where `::placeholder` is defined all the others aren't really necessary, we use a module called `autoprefixer` that transforms `::placeholder {}` in the various browser specific versions so they can be removed